Orange/widgets/data/tests/test_owtable.py

```diff
@@ -247,6 +247,9 @@ def setUpDB(self):
         # pylint: disable=attribute-defined-outside-init
         conn, iris = self.create_iris_sql_table()
         data = SqlTable(conn, iris, inspect_values=True)
+        if self.current_db == "mssql":
+            # when loading data from mssql db, Sql widget return Table (not SqlTable)
```

Comment:
return -> returns
(should be fixed in all 3 instances in the PR)

Reply:
Done. :)